modules/phase_field/src/userobjects/EBSDReader.C

```diff
@@ -262,11 +262,17 @@ EBSDReader::indexFromPoint(const Point & p) const
   unsigned int x_index, y_index, z_index, global_index;
 
   x_index = (unsigned int)((p(0) - _minx) / _dx);
+  if (x_index == _nx)
+    --x_index;
```

Comment:
Umm... We'll have to do something about this. The EBSDReader can only return one grain per point.